third_party/opencv/opencv_cmake_flags.txt

```diff
@@ -1,13 +1,17 @@
--D BUILD_LIST=core,improc,imgcodecs,calib3d,features2d,highgui,imgproc,video,videoio,optflow \
--D OPENCV_EXTRA_MODULES_PATH=/opt/opencv_contrib_repo/modules
--D CMAKE_BUILD_TYPE=Release \
--D CMAKE_INSTALL_PREFIX=/opt/opencv \
--D OPENCV_LIB_INSTALL_PATH=lib \ 
--D CV_ENABLE_INTRINSICS=ON \
--D WITH_EIGEN=ON \
--D WITH_PTHREADS=ON \
--D WITH_PTHREADS_PF=ON \
--D WITH_JPEG=ON \
--D WITH_PNG=ON \
--D WITH_TIFF=ON
-
+-DBUILD_LIST=core,improc,imgcodecs,calib3d,features2d,highgui,imgproc,video,videoio,optflow \
+-DOPENCV_EXTRA_MODULES_PATH=/opt/opencv_contrib_repo/modules
+-DCMAKE_BUILD_TYPE=Release \
+-DCMAKE_INSTALL_PREFIX=/opt/opencv \
+-DOPENCV_LIB_INSTALL_PATH=lib \ 
+-DBUILD_TESTS=OFF -DBUILD_PERF_TESTS=OFF -DBUILD_opencv_ts=OFF \
```

Comment:
Why are you changing convention from "-D FLAGNAME" to "-DFLAGNAME and additionally only for some parameters keep the rule one flag per line?